docs/overview/workflows-101.rst

```diff
@@ -2,4 +2,38 @@ Workflows 101
 =============
 
 .. meta::
-    :description lang=en: Introduction to workflows.
\ No newline at end of file
+    :description lang=en: Introduction to workflows.
+
+What is a workflow?
+-------------------
+
+According to `Merriam-Webster <https://www.merriam-webster.com/dictionary/workflow/>`_, a workflow is
+
+    the sequence of steps involved in moving from the beginning to the end of a working process
+
+For our purposes, we'll attempt to more specifically define "scientific workflows" (or `#sciworkflows <https://twitter.com/search?q=%23sciworkflows&src=typd/>`_ for Twitter users). Let's start by clarifying what we *don't* mean by scientific workflows (henceforth "workflows" for short):
+
```

Comment:
Computational workflows arose due to the need of reproducible computational analysis. Often times scientists had many steps in a particular analysis protocol and needed to execute those same tasks thousands of times. As such, building tools to do this for them has become necessary. "Workflows" as we know them today are simply a framework by which these custom automation scripts can be abstracted.